### ISSUE TITLE
UnicodeDecodeError when username contains non-ascii characters

### ISSUE THREAD

**Issue by jaraco** [jeffwidman/bitbucket_issue_migration#19]
```
> py -2 .\migrate.py kang python-keyring-lib jaraco jaraco/keyring
Please enter your GitHub password

Traceback (most recent call last):
  File ".\migrate.py", line 304, in <module>
    push_issue(gh_username, gh_repository, issue, body, comments)
  File ".\migrate.py", line 266, in push_issue
    format_comment(comment),
  File ".\migrate.py", line 122, in format_comment
    comment['user'].encode('utf-8')
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe2 in position 153: ordinal not in range(128)
```

**Comment by TurkeyMan** [comment]
+1

**Comment by henhouse** [comment]
+1

**Comment by rporrini** [comment]
+1

**Comment by jbkunst** [comment]
+1

**Comment by jeffwidman** [comment]
I think this was fixed by 8384492d436d868481f031398a75530b96dd949e and 496a53e9c9ca96f734d0b6e5bafbd3f758ec4b67
